db/migrate/20170126214854_create_bet_managers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBetManagers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :bet_managers do |t|
+      t.integer :user_id
```

Comment:
This is correct, Rails also has the `#references` and `#belongs_to` migration methods which could be useful

Generally we include indexes and foreign key declarations when we are creating references.
https://tomafro.net/2009/08/using-indexes-in-rails-index-your-associations
https://robots.thoughtbot.com/referential-integrity-with-foreign-keys